pyproject.toml

```diff
@@ -45,7 +45,8 @@ pygithub = "^2.3.0"
 jupyterlab-spellchecker = "^0.8.4"
 
 [tool.poetry.scripts]
-process_repo_entropy = "almanack.cli:process_repo_entropy"
+almanack = "almanack.cli:check"
```

Comment:
It looks like this might not link to the module accurately (`cli` is under the `reporting` package). Consider updating this to help reflect things and testing your command line usage with `poetry run almanack ...`.